src/main/dialogs.ts

```diff
@@ -17,6 +17,10 @@ export function setupDialogs() {
     showConfirmationDialog(args);
   });
 
+  ipcMainManager.on(IpcEvents.SHOW_LOCAL_VERSION_FOLDER_DIALOG, async (_event) => {
+    await showOpenDialog(_event);
```

Comment:
This should probably be `event` if we actually forward it (TypeScript uses the `_` to denote "I don't use this")